workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGenerationLib/CodeWriter/Scopes/Abstract/ScopeBody.cs

```diff
@@ -123,14 +123,14 @@ public void Return(string toReturn)
             Line($"return {toReturn};");
         }
 
-        public void ProfilerStart(string name)
+        public void ProfileScope(string markerVariableName, Action<CustomScopeBlock> populate)
```

Comment:
Need to update the codegen docs where we have examples of using these 

https://github.com/spatialos/gdk-for-unity/blob/feature/profile-markers/workers/unity/Packages/io.improbable.gdk.tools/.CodeGenDocs/3-using-the-codewriter.md#profiler-sampling